### PR TITLE
CTOR-1055 : Refresh cloud-aws-ebs

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh cloud-aws-ebs

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-ebs.md [hunk 1]
```diff
@@ -5,71 +5,85 @@ title: Amazon EBS
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+## Dépendances du Connecteur de supervision
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-ebs.md:8]:
```suggestion

Amazon Elastic Block Store (EBS) est un service de stockage par bloc hautes performances et simple d'utilisation conçu en vue d'une utilisation avec Amazon Elastic Compute Cloud (EC2) 
pour les charges de travail exigeantes en débit et en transactions à n'importe quelle échelle.
Des charges de travail variées, telles que des bases de données relationnelles et non relationnelles, des applications d'entreprise, des applications conteneurisées, des moteurs d'analyse Big Data, 
des systèmes de fichiers et des workflows multimédias, sont largement déployées sur Amazon EBS.

## Dépendances du Connecteur de supervision
```

File: pp/integrations/plugin-packs/procedures/cloud-aws-ebs.md [hunk 2]
```diff
@@ -5,72 +5,84 @@ title: Amazon EBS
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+## Connector dependencies
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/cloud-aws-ebs.md:8]:
```suggestion

Amazon Elastic Block Store (Amazon EBS) provides block level storage volumes for use with EC2 instances. 
EBS volumes behave like raw, unformatted block devices. You can mount these volumes as devices on your instances. 
You can mount multiple volumes on the same instance, and you can mount a volume to multiple instances at a time. 
You can create a file system on top of these volumes, or use them in any way you would use a block device (like a hard drive). 
You can dynamically change the configuration of a volume attached to an instance.

## Connector dependencies
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-ebs.md [hunk 3]
```diff
@@ -6,70 +6,90 @@ import Tabs from '@theme/Tabs';
...
 
-## Vue d'ensemble
-
 Amazon Elastic Block Store (EBS) est un service de stockage par bloc hautes performances et simple d'utilisation conçu en vue d'une utilisation avec Amazon Elastic Compute Cloud (EC2) 
 pour les charges de travail exigeantes en débit et en transactions à n'importe quelle échelle.
 Des charges de travail variées, telles que des bases de données relationnelles et non relationnelles, des applications d'entreprise, des applications conteneurisées, des moteurs d'analyse Big Data, 
 des systèmes de fichiers et des workflows multimédias, sont largement déployées sur Amazon EBS.
 
-Le Plugin Centreon Amazon EBS utilise l'API Amazon Cloudwatch pour collecter les métriques associées.
+## Dépendances du Connecteur de supervision
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-ebs.md:14]:
```suggestion
## Dépendances du connecteur de supervision
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-ebs.md [hunk 4]
```diff
@@ -78,19 +98,15 @@ https://docs.aws.amazon.com/fr_fr/AWSEC2/latest/UserGuide/using_cloudwatch_ebs
 
 ### Privilèges AWS
 
-Voici la liste des droits nécessaires au travers des *access/secret keys* utilisées pour pouvoir implémenter la supervision des ressources Amazon EBS: 
-
-| AWS Privilege                  | Description                                                     |
-| :----------------------------- | :-------------------------------------------------------------- |
-| ec2:DescribeVolumes            | Describes the specified EBS volumes or all of your EBS volumes  |
-| cloudwatch:getMetricStatistics | Get metrics from the AWS/EBS namespace on Cloudwatch            |
+Configurez un compte de service (via une combinaison d'access et secret key) et affectez-lui les privilèges suivants :
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-ebs.md:101]:
```suggestion
Configurez un compte de service (via une combinaison d'access key et secret key) et affectez-lui les privilèges suivants :
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-ebs.md [hunk 5]
```diff
@@ -103,129 +119,344 @@ yum install perl-Paws
...
-	--timeframe='600' \
-	--period='60' \
-	--warning-volume-write-bytes='60000000' \
-	--critical-volume-write-bytes='90000000' \
-	-- verbose
+	--aws-secret-key='' \
+	--aws-access-key='' \
+	--aws-role-arn='' \
+	--region='' \
+	--volume-id='' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-ebs.md:306]: ?

File: pp/integrations/plugin-packs/procedures/cloud-aws-ebs.md [hunk 6]
```diff
@@ -102,127 +119,344 @@ yum install perl-Paws
...
-	--timeframe='600' \
-	--period='60' \
-	--warning-volume-write-bytes='60000000' \
-	--critical-volume-write-bytes='90000000' \
-	-- verbose
+	--aws-secret-key='' \
+	--aws-access-key='' \
+	--aws-role-arn='' \
+	--region='' \
+	--volume-id='' \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-aws-ebs.md:306]: ?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-ebs.md [hunk 7]
```diff
@@ -103,129 +119,344 @@ yum install perl-Paws
...
-	--statistic='average' \
-	--timeframe='600' \
-	--period='60' \
-	--warning-volume-write-bytes='60000000' \
-	--critical-volume-write-bytes='90000000' \
-	-- verbose
+	--aws-secret-key='' \
+	--aws-access-key='' \
+	--aws-role-arn='' \
+	--region='' \
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-ebs.md:305]:
```suggestion
	--aws-secret-key='XXXX' \
	--aws-access-key='XXXX' \
	--aws-role-arn='' \
	--region='eu-west-1' \
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-ebs.md [hunk 8]
```diff
@@ -103,129 +119,344 @@ yum install perl-Paws
...
+| AWSACCESSKEY  | Set AWS access key                                                                                                                                 |                   |             |
+| AWSASSUMEROLE | Set Amazon Resource Name of the role to be assumed                                                                                                 |                   |             |
+| AWSCUSTOMMODE | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option                         | awscli            |             |
+| AWSREGION     | Set the region name (required)                                                                                                                     |                   | X           |
+| AWSSECRETKEY  | Set AWS secret key                                                                                                                                 |                   |             |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-ebs.md:231]:
```suggestion
| AWSACCESSKEY  | Set AWS access key                                                                                                                                 |                   | X           |
| AWSASSUMEROLE | Set Amazon Resource Name of the role to be assumed                                                                                                 |                   |             |
| AWSCUSTOMMODE | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option                         | awscli            |             |
| AWSREGION     | Set the region name (required)                                                                                                                     |                   | X           |
| AWSSECRETKEY  | Set AWS secret key                                                                                                                                 |                   | X           |
```

File: pp/integrations/plugin-packs/procedures/cloud-aws-ebs.md [hunk 9]
```diff
@@ -102,127 +119,344 @@ yum install perl-Paws
...
+|:--------------|:-----------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| AWSACCESSKEY  | Set AWS access key                                                                                                                       |                   |             |
+| AWSASSUMEROLE | Set Amazon Resource Name of the role to be assumed                                                                                       |                   |             |
+| AWSCUSTOMMODE | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option               | awscli            |             |
+| AWSREGION     | Set the region name (required)                                                                                                           |                   | X           |
+| AWSSECRETKEY  | Set AWS secret key                                                                                                                       |                   |             |
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/cloud-aws-ebs.md:233]:
```suggestion
| AWSACCESSKEY  | Set AWS access key                                                                                                                       |                   | X           |
| AWSASSUMEROLE | Set Amazon Resource Name of the role to be assumed                                                                                       |                   |             |
| AWSCUSTOMMODE | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option               | awscli            |             |
| AWSREGION     | Set the region name (required)                                                                                                           |                   | X           |
| AWSSECRETKEY  | Set AWS secret key                                                                                                                       |                   | X           |
```

File: pp/integrations/plugin-packs/procedures/cloud-aws-ebs.md [hunk 10]
```diff
@@ -102,127 +119,344 @@ yum install perl-Paws
...
-	--statistic='average' \
-	--timeframe='600' \
-	--period='60' \
-	--warning-volume-write-bytes='60000000' \
-	--critical-volume-write-bytes='90000000' \
-	-- verbose
+	--aws-secret-key='' \
+	--aws-access-key='' \
+	--aws-role-arn='' \
+	--region='' \
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/cloud-aws-ebs.md:305]:
```suggestion
	--aws-secret-key='XXXX' \
	--aws-access-key='XXXX' \
	--aws-role-arn='' \
	--region='eu-west-1' \
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**